run_dir/static/js/pricing_main.js

```diff
@@ -711,25 +711,19 @@ app.component('v-products-table', {
         <thead class="table-light">
           <tr class="sticky">
             <th v-if="quotable">Quoting</th>
-            <th class="sort" data-sort="id">ID</th>
             <th class="sort" data-sort="category">Category</th>
             <th class="sort" data-sort="type">Type</th>
             <th class="sort" data-sort="name">Name</th>
-            <th class="sort" data-sort="components">Components</th>
```

Comment:
Hmm, I think this would remove it from all the cost calculator tables right? I agree that it probably doesn't need to be on the agreements page

Reply:
Yes, it removes it from everywhere that uses this component. I'm guessing we'll have to make a different component or else generate the columns dynamically if we want to keep using the same component for this?

Reply:
@alneberg Added it back to the new cost calculator tables now.